fusor-ember-cli/app/components/pf-modal-base.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+    openCloseModal: Ember.observer('openModal', function() {
+        if (this.get('openModal')) {
+            return Ember.$('#'+this.get('idModal')).modal('show');
+        } else {
+            return Ember.$('#'+this.get('idModal')).modal('hide');
+        }
+    }),
+
+    actions: {
+        closeModal: function() {
+           return this.set('openModal', false);
```

Comment:
Actions shouldn't return values unless it's trying to control the bubbling behavior: https://guides.emberjs.com/v1.13.0/templates/actions/#toc_action-bubbling

Is this the intention here? I'm not actually sure what `this.set` returns.